contributors/design-proposals/storage/csi-migration.md

```diff
@@ -287,8 +287,29 @@ existing Pods in the ADC.
 
 
 ### Volume Resize
-
-TODO: Design
+#### Offline Resizing
+In the OperationGenerator, `GenerateExpandVolumeFunc` is used to expand a volume after 
+Pod is deleted. At the beginning of the `GenerateExpandVolumeFunc`, we will check whether 
+the volume supports migration and whether migration is enabled. If enabled, we will return
+a noop function to by-pass in-tree resizer (expand controller).
```

Comment:
do we still need to annotate the PVC appropriately in this step?

Reply:
We do. We should be updating this document soon.